shared/index.d.ts

```diff
@@ -595,10 +606,18 @@ declare interface Movie extends Panel {
 }
 
 /** An interactive number input.
- * @todo These types are incomplete and unverified!
+ * Values are all integers in C++ for some reason.
+ * Has a onvaluechanged event, I don't understand our event system!
```

Comment:
"I don't understand our event system!" doesn't really give a lot of helpful information.
If you want to keep it, add a `@todo` (and initials, since it's in the first person.)

Reply:
Oh haha, good spot. Didn't mean to commit this, had left this as a personal note to clean up when I committed, and completely missed it. Totally agree, have updated the comment!